.github/workflows/bazel-cache-push.yml

```diff
@@ -82,6 +82,16 @@ jobs:
 
           aws s3 cp ${{ env.BAZEL_CACHE_MAGMA_VM_TAR }} ${{ env.S3_BUCKET_PATH }}/${{ env.BAZEL_CACHE_MAGMA_VM_TAR}}
           aws s3 cp ${{ env.BAZEL_CACHE_REPO_MAGMA_VM_TAR }} ${{ env.S3_BUCKET_PATH }}/${{ env.BAZEL_CACHE_REPO_MAGMA_VM_TAR}}
+      - name: Notify failure to slack
+        if: failure()
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_BAZEL_CI }}
+          SLACK_TITLE: "VM Caches"
+          SLACK_USERNAME: "Push Bazel Cache To S3"
```

Comment:
There seems to be information duplication in those two entries. I would not see a reason, why the title and the username should be changed to something like a title.

Reply:
> To me it seems as if `Ref` and `Event` do not provide sensible information in the slack message. We might consider removing them by:
> 
> ```yaml
> MSG_MINIMAL: actions,commit
> ```

Done.